### PR TITLE
POC with socketlib

### PR DESCRIPTION
**Pull request by vit0r**
POC

### CODE REVIEW

File: poc_ports_check.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+from sys import argv
+from socket import socket, setdefaulttimeout, AF_INET, SOCK_STREAM
+
+*, timeout, host, port_check = argv
```
Comment (rafaellott) [inline poc_ports_check.py]: I got SyntaxError for this '*' on python 2.7 and python 3.6.
Is not better to change to _?

Comment (vit0r) [reply poc_ports_check.py]: yeap!

**Comment by rafaellott** [comment]
@vit0r I added the shebang to make the file executable by shell

**Comment by vit0r** [comment]
Nice :smile:

**Pull request merged**